packages/hoppscotch-common/src/components/environments/teams/index.vue

```diff
@@ -159,11 +178,20 @@ const props = defineProps<{
   loading: boolean
 }>()
 
-// Sort environments alphabetically by default
+const filterTexts = ref("")
```

Comment:
`filterText` would suit the context better, given that the query points to a single entity/resource. However, the results can have multiple entries accommodating entries with the same name.

```suggestion
const filterText = ref("")
```